debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -379,12 +394,22 @@ public String getRealDatabaseName() {
     private ZoneId retrieveTransactionTimezone(boolean supportsAtTimeZone) {
         final String serverTimezoneConfig = config().getString(SERVER_TIMEZONE_PROP_NAME);
 
-        if (!supportsAtTimeZone && serverTimezoneConfig == null) {
-            LOGGER.warn("The ' " + SERVER_TIMEZONE_PROP_NAME
-                    + "' option should be specified to avoid incorrect timestamp values in case of different timezones between the database server and this connector's JVM.");
+        if (supportsAtTimeZone) {
+            if (serverTimezoneConfig != null) {
+                LOGGER.warn("The '{}' option should not be specified with SQL Server 2016 and newer");
+            }
+        }
+        else {
+            if (serverTimezoneConfig == null) {
+                LOGGER.warn("The ' " + SERVER_TIMEZONE_PROP_NAME
```

Comment:
You should make log messages consistent, either both using slf4j placeholders or bot using string concats but not both

Reply:
Good catch on both. I'll adjust and merge. Thanks for reviewing, @jpechane!